src/services/defaultActionsImport.js

```
@@ -0,0 +1,269 @@
+ */
+function transformActionsToCustomTiles(actions, locale = 'en', gameMode = 'online') {
+  const customTiles = [];
+
+  Object.entries(actions).forEach(([groupKey, groupData]) => {  
+    // Skip if there are no actions or if it's just a label
+    if (!groupData.actions || Object.keys(groupData.actions).length === 0) {
+      return;
+    }
+
+    // Process each intensity level
+    Object.entries(groupData.actions).forEach(([_, actionsList], intensityIndex) => {
```

ESLint

[error] '_' is defined but never used.

(no-unused-vars)